src/chat/entity_button.rs

```diff
@@ -1,9 +1,10 @@
+use crate::data::chats::chat_entity::{ChatEntityId, ChatEntityRef};
```

Comment:
This was `agent_button.rs`. Modified to be able to handle both, agents and models.

Also, removed the flags used to change the internal behavior of the widget and removed the globally dispatched actions caused by those flags. Instead, a simply `clicked()` method has been added, and handling occurs on the parent maximizing reusability and simplifying the internals.